packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js

```diff
@@ -115,9 +117,14 @@ class AccountName extends Component {
       createAccountStore: { address, name },
       error,
       history,
-      location: { pathname }
+      location: { pathname },
+      accountsInfo
     } = this.props;
+
     const currentStep = pathname.slice(-1);
+    const accountNameExists = !!Object.values(accountsInfo).find(
+      info => info.name === name
```

Comment:
I would even go further and perform a case unsensitive matching
```suggestion
      info => info.name.toLowerCase() === name.toLowerCase()
```